apps/omg_api/mix.exs

```diff
@@ -47,7 +47,8 @@ defmodule OMG.API.MixProject do
       {:omg_db, in_umbrella: true},
       {:omg_eth, in_umbrella: true},
       {:omg_rpc, in_umbrella: true},
-      {:sentry, "~> 6.2.0"}
+      {:sentry, "~> 6.2.0"},
+      {:appsignal, "~> 1.0"}
```

Comment:
likewise, the `mix.exs` entry in `deps` looks like it could be entered just once in top-level `mix.exs`